pom.xml

```diff
@@ -31,14 +31,30 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk</artifactId>
-      <version>1.11.478</version>
+      <version>1.12.395</version>
```

Comment:
As our friend Nikhil originally suggested, it may be more-prudent to prune down the dependencies imported so that, instead of importing the whole aws-java-sdk, to only import the latest version of aws-java-sdk-devicefarm and aws-java-sdk-core

Reply:
Good suggestion, I will look at doing this